cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShippingStepPage {
+  private proceedToCheckoutBtn: string;
+  private checkBox: string;
+
+  constructor() {
+    this.checkBox = "[type='checkbox']";
+    this.proceedToCheckoutBtn = ".cart_navigation button";
+  }
+
+  public clickCheckBox(): void {
+    cy.get(this.checkBox).click();
+  }
+
+  public clickProceedToCheckout(): void {
+    cy.get(this.proceedToCheckoutBtn).click();
+  }
+}
+
+export default new ShippingStepPage();
```

Comment:
not sure about this approach, I prefer the one suggested by the guide, export the class and then have a page/index.ts so you can import from the page index in the buy-shirt.cy.ts file